source/reference/supported-server-version.txt

```diff
@@ -0,0 +1,83 @@
+.. _c2c-server-version-compatibility:
+
+================================================
+MongoDB Server Version Compatibility and Support
+================================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Before you run {+c2c-product-name+}, consider the following MongoDB server 
+version limitations and requirements: 
+
+- .. include:: /includes/fact-minimum-server-version-support.rst
+
+- ``mongosync`` doesn't support MongoDB rapid releases, such as 6.1 or 6.2.
```

Comment:
[Suggestion]

This isn't net new content, but it might help to update to the current rapid releases: 7.1 and 7.2.

Reply:
Ok, will backport. Thanks!